tests/python/gaia-ui-tests/gaiatest/tests/functional/lockscreen/test_lockscreen_unlock_to_camera_with_passcode.py

```diff
@@ -17,7 +17,8 @@ def setUp(self):
         # Turn off geolocation prompt
         self.apps.set_permission('Camera', 'geolocation', 'deny')
 
-        self.data_layer.set_setting('lockscreen.passcode-lock.code', self._input_passcode)
+        self.setPasscodeTo1337()
```

Comment:
I'm fine with the value being part of the method name. If we need other values in the future we can compute them and refactor.